index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;700&family=Poppins:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles/base.css" />
+    <link rel="stylesheet" href="styles/header.css" />
+    <link rel="stylesheet" href="styles/banner.css" />
+    <link rel="stylesheet" href="styles/section.css" />
+    <link rel="stylesheet" href="styles/footer.css" />
+    <!-- 반응형 : 타블렛, 핸드폰 지원 -->
+    <link rel="stylesheet" href="styles/tablet.css" />
+    <link rel="stylesheet" href="styles/phone.css" />
```

Comment:
💊 제안
css를 목적별로 분리해주신 점 좋습니다~
다만 이렇게 되면 해당 파일에서 불러와야 하는 css 파일이 너무 많아지니 아래와 같은 방식을 추천드려요.

```suggestion
    <link rel="stylesheet" href="styles/base.css" />
    <link rel="stylesheet" href="styles/index.css" /> 
```

페이지에 대응하는 css 파일을 만들고 내부적으로 해당 페이지 구현을 위해 필요한 css 들을 import하는 방식입니다.
이렇게 하시면 index.html 은 필요한 파일만 가지게 되고 관리측면에서도 편리합니다.